projects/36/bhxiv/paper.md

```diff
@@ -103,7 +103,7 @@ pasting above link (or yours) in
+The COVID-19 crisis demonstrates a critical requirement for rapid and efficient sharing of data to facilitate the global response to this and future pandemics. We can address this challenge by making viral genomic and patient phenomic data FAIR, and formalising it to permit seamless data integration for facilitating data analysis. Phenopackets is a standard file format for sharing phenotypic information that facilitates communication within the research and clinical genomics communities [@jacobsen2022a]. The Observational Medical Outcomes Partnership (OMOP) Common Data Model allows for large-scale analysis of distributed data to generate evidence for research that promotes better health decisions and better care [@overhage2012a]. These gathered data is used by epidemiologists to monitor the infection, model it and make outbreak analysis and predictions to evaluate policy interventions. To harness Machine Learning (ML) and Artificial Intelligence (AI) approaches to discover meaningful patterns in epidemic outbreaks, we need to ensure that data are FAIR, i.e. data and metadata are accessible and actionable by machines. To leverage data for federated learning/analytics, datasets can be discovered in FAIR Data Points; FAIR data repositories that publish human- and machine-readable metadata for data resources. This project aims to enhance interoperability between health and research data by mapping Phenopackets and OMOP and representing COVID-19 metadata using the FAIR principles to enable discovery, integration and analysis of genotypic and phenotypic data.
 
```

Comment:
Hi @ljgarcia thank you so much for your review and PR. Is it correct that in this paragraph you only did one change which is: 'The OMOP model' to 'The Observational Medical Outcomes Partnership (OMOP) Common Data Model'? If yes, this is correct and a good call! is it possible for you to add the acronym for common data model such as 'The Observational Medical Outcomes Partnership (OMOP) Common Data Model (CDM)'? In that way we are consistent. Then I will accept your suggestion. Thank you!

Reply:
It was OMOP model --> changed to Observational Medical Outcomes Partnership (OMOP) Common Data Model

Reply:
Thanks, I will add also the CDM acronym.